cli/src/test/resources/xmlmerger/intellijJdkTable/template.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<application xmlns:merge="https://github.com/devonfw/IDEasy/merge" merge:strategy="combine">
+  <component name="ProjectJdkTable">
+    <jdk version="2" merge:id="./jdk/version[@value='OpenJDK $[JAVA_VERSION]']/.." merge:strategy="override">
```

Comment:
for `ide-settings` I rather suggest `./jdk/version[@name='Java']/..` but for the tests I also wanted to cover and verify #498